pkg/helper/workloads.go

```diff
@@ -0,0 +1,152 @@
+package helper
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+)
+
+// ApplyYamlInFolder reads a folder and attempts to create objects in K8s with the yaml
+func ApplyYamlInFolder(folder, namespace string, kube kubernetes.Interface) ([]runtime.Object, error) {
+	var (
+		objects []runtime.Object
+		obj     runtime.Object
+		files   []string
+		err     error
+	)
+	pwd, _ := os.Getwd()
+	err = filepath.Walk(pwd+folder, func(path string, info os.FileInfo, err error) error {
+		if strings.Contains(path, ".yaml") || strings.Contains(path, ".yml") {
```

Comment:
nits: Use info.Name() instead of path, and use strings.HasSuffix() instead of strings.Contains(). Unlikely to cause issues, but reduces the surface area for complication.